src/app.js

```diff
@@ -1 +1,18 @@
 'use strict';
+
+const fs = require('fs');
+
+function coppyFile(fileToCopy, newFile) {
+  if (fileToCopy === newFile) {
+    return;
+  }
+
+  const dataToWrite = fs.readFileSync(fileToCopy);
+
+  fs.writeFileSync(newFile, dataToWrite);
```

Comment:
Sorry, didn't notice this before:

You are reading the **entire** file into memory before starting to copy here. This is very inefficient if you copy a large file.

Use streams to avoid this